### PR TITLE
[Expo-Zustand-Styled-Components] feat: Homepage

### PR DESCRIPTION
**Pull request by markshenouda**
Web view
Screenshot 2023-03-08 at 3 21 39 PM

iOS
Screenshot 2023-03-08 at 3 21 51 PM

### CODE REVIEW

File: expo-zustand-styled-components/src/components/RepoCard/RepoCard.tsx [hunk 1]
```diff
@@ -14,33 +14,34 @@ import { StarLogo } from '../Icons/StarLogo';
...
+  updated_at,
+  star,
+}: RepoCardProps) => {
   return (
-    <Card key={id}>
+    <Card>
       <Content>
         <Heading>
-          <Link>Repo name here</Link>
+          <Link>{full_name}</Link>
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/RepoCard/RepoCard.tsx:42]: On press of this, it should take use to the repo page, currently, there is nothing addressing that.
What do you plan?

**Review by hdJerry — CHANGES_REQUESTED** [review]
This is really nice work, I just have a request here.

File: expo-zustand-styled-components/src/components/RepoCard/RepoCard.tsx (hunk 1 above)
Comment (markshenouda) [reply expo-zustand-styled-components/src/components/RepoCard/RepoCard.tsx:42]: The page doesn't exist now, I'll add it to the repos page ticket

**Review by hdJerry — APPROVED** [review]

**Review by Megio — APPROVED** [review]
LGTM

**Review by AllStackDev1 — APPROVED** [review]

**Pull request merged**